CondTools/SiStrip/test/SiStripApvSimulationParametersBuilder_cfg.py

```diff
@@ -15,7 +15,7 @@
         authenticationPath = cms.untracked.string('/afs/cern.ch/cms/DB/conddb')
     ),
     timetype = cms.untracked.string('runnumber'),
-    connect = cms.string('sqlite_file:dbfile.db'),
+    connect = cms.string('sqlite_file:SiStrpConditionsDBFile.db'),
```

Comment:
Did you intentionally skip the "i"?
```suggestion
    connect = cms.string('sqlite_file:SiStripConditionsDBFile.db'),
```
?